packages/vite-config/src/portals-vite-config.ts

```diff
@@ -0,0 +1,83 @@
+import vitestConfig from './vitest-config.js'
+import { mergeConfig } from 'vitest/config'
+import { createHtmlPlugin } from 'vite-plugin-html'
+
+export default mergeConfig(vitestConfig, {
+  plugins: [
+    createHtmlPlugin({
```

Comment:
The HTML for each portal is identical. All of the shared content is injected via template (see `headContent`, `gtmNoscript`)